MobileCenterAnalytics/MobileCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -1,8 +1,3 @@
-#import <Foundation/Foundation.h>
```

Comment:
Yeah.. This is a bad merge. I also fixed in the branch of feature/support-macos-10-9.